pkg/loadbalancer/loadbalancer.go

```diff
@@ -315,6 +315,36 @@ func IsValidStateTransition(old, new BackendState) bool {
 	return true
 }
 
+func GetBackendState(state string) (BackendState, error) {
+	switch strings.ToLower(state) {
+	case "active", "":
+		return BackendStateActive, nil
+	case "terminating":
+		return BackendStateTerminating, nil
+	case "quarantined":
+		return BackendStateQuarantined, nil
+	case "maintenance":
+		return BackendStateMaintenance, nil
+	default:
+		return BackendStateInvalid, fmt.Errorf("invalid backend state %s", state)
+	}
+}
+
+func GetBackendStateString(state BackendState) (string, error) {
```

Comment:
this maybe a little nicer implemented on the `BackendState` type itself. (Same as GetBackendState)